helm/aws-operator/values.yaml

```diff
@@ -1,4 +1,16 @@
-namespace: giantswarm
-
-userID: 1000
-groupID: 1000
+docker:
+  image: "quay.io/giantswarm/aws-operator:[[ .DockerTag ]]"
```

Comment:
This is in prep for china registry.

Reply:
This is nice but can we structure it more so we have a separate value for the registry?

This is how we do it for the core components.

https://github.com/giantswarm/kubernetes-coredns/blob/3ee66271fdffde581897f1ee42fe9d4824e3da93/helm/kubernetes-coredns-chart/values.yaml#L22-L25

Reply:
We should use `[[ .Version ]]` here.

Reply:
At the time of creating the PR we didn't even have push-to-docker job in the orb. I need to merge latest changes.

Reply:
@rossf7 I changed that accordingly.